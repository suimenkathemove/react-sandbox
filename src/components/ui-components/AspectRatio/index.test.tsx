import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import { AspectRatio } from './';

const setup = () => render(<AspectRatio {...defaultProps} />);

describe('AspectRatio', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
