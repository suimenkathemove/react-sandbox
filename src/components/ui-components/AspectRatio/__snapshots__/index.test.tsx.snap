// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AspectRatio render 1`] = `
<DocumentFragment>
  <div
    class="outer"
    style="padding-bottom: 75%;"
  >
    <div
      class="inner"
    >
      <img
        alt="random"
        src="https://picsum.photos/200"
        style="object-fit: cover;"
      />
    </div>
  </div>
</DocumentFragment>
`;
